docs/bokeh/source/conf.py

```diff
@@ -25,14 +25,14 @@
 version = settings.docs_version() or __version__
 
 bokeh_example_dirs = [
-    'examples/plotting/file',
-    'examples/models/file',
+    'examples/plotting',
+    'examples/models',
```

Comment:
This is a good start, but in future I would like even more reorganization. In particular most examples under `models/` are only using model API, because there is no alternative, and not really showing any low-level behavior as those examples were meant to show. There is really only handful of examples that really belong there. Everything else should be under `plotting/` or `embed/`. `plotting/` has currently 110+ examples, so that would also require better structure as well. I will start a discussion for this.

Reply:
For sure, one of the motivations to flatten things here is to afford more head room for new subdivisions, esp. under `plotting`.